corehq/apps/builds/models.py

```diff
@@ -239,7 +239,7 @@ def patch_release(self):
     def release_greater_than_or_equal_to(self, version):
         if not self.version:
             return False
-        return StrictVersion(self.version) >= StrictVersion(version)
+        return LooseVersion(self.version) >= LooseVersion(version)
```

Comment:
Same as previous comment.

Reply:
see previous response

Reply:
This comparison will be equivalent with `LooseVersion` and `StrictVersion`. The only thing with `StrictVersion` is that if the version is invalid it would have thrown a `ValueError` which we don't appear to catch here anyway.